composer.json

```diff
@@ -39,9 +39,7 @@
     "laravel/tinker": "^2.0",
     "mockery/mockery": "^1.0",
     "nunomaduro/collision": "^3.0",
-    "phpunit/phpunit": "^8.0",
-    "symfony/css-selector": "3.1.*",
-    "symfony/dom-crawler": "3.1.*"
```

Comment:
These were old and were removed back in Laravel 5.4, but I think we forgot to remove!